pmd-apex/src/main/resources/category/apex/performance.xml

```diff
@@ -80,4 +80,26 @@ public class Something {
         </example>
     </rule>
 
+    <rule name="AvoidDatabaseMethodCallsInLoops"
```

Comment:
We try to order the entries alphabetically by rule name. This rule needs to be added as the first rule in the file....